IDR/production/1.compute_statistics.py

```diff
@@ -79,10 +88,10 @@ def nested_dict_pairs_iterator(elements_and_counts_dict):
         data=unique_elements_and_counts_list,
         columns=["Study", "Screen", "Attribute", "Element", "Count"],
     )
-    
-    unique_elements_and_counts_df["Element"] = unique_elements_and_counts_df["Element"].apply(
-            lambda x: str(x)
-        )
+
+    unique_elements_and_counts_df["Element"] = unique_elements_and_counts_df[
+        "Element"
+    ].apply(lambda x: str(x))
```

Comment:
Consider using `astype` here to increase readability.

```suggestion
    ].astype("str")
```

Reply:
@d33bs Ah I didn't know you could do that for pandas. I changed it to `astype("str")`.

Added in e638f2c.